docs/reference/commands/reset-password.asciidoc

```diff
@@ -59,12 +59,17 @@ option. For more information about debugging connection failures, see
 
 `-u, --username`:: The username of the native realm user or built-in user.
 
+`--url`:: Specifies the base URL (hostname and port of the local node) that the tool uses to submit API
+requests to {es}. The default value is determined from the settings in your
+`elasticsearch.yml` file. If `xpack.security.http.ssl.enabled` is set to `true`,
```

Comment:
It's not that simple actually. It can be `network.host` or `http.host` , with `http.port`, let alone [advanced options](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-network.html#advanced-network-settings) or talking about `_site_`, `_local_`, `_global_` etc. I don't think this is the place to go through all these in detail :/

Reply:
Of course it's not that simple 😆 , which is probably why you opted for simplicity in your original description. I'm 👍 with keeping your original text -- thank you for the explanation!